docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -3,14 +3,16 @@ This manual is intended for users that have knowledge about the structure of the
 
 When loading studies into cBioPortal it is possible for warnings to occur that are caused by an outdated seed database. Gene symbols can be deprecated or be assigned to a different Entrez Gene in a new release. Also Entrez Gene IDs can be added. This markdown explains how to update the seed database, in order to use the most recent Entrez Gene IDs.
 
-The cBioPortal scripts package provides a method to update the `gene` and `gene_alias` tables. This requires the latest version of the NCBI Gene Info.
+The cBioPortal scripts package provides a method to update the `gene` and `gene_alias` tables.
+
+## Prepare
 
 ### Human genes
-Homo_sapien.gene_info.gz\
-ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Homo_sapiens.gene_info.gz
+Download [gene_info.txt](https://github.com/cBioPortal/datahub-study-curation-tools/blob/master/gene-table-update/build-input-for-importer/gene_info.txt)  
+Generated based on latest HGNC release using script [HERE](https://github.com/cBioPortal/datahub-study-curation-tools/tree/master/gene-table-update/build-input-for-importer) 
 
 ### Mouse genes
-Mus_musculus.gene_info.gz\
+Download `Mus_musculus.gene_info.gz` from   
```

Comment:
So we are telling users that for human they use our own distribution of hgnc genes with the -hgnc argument, but for mice they use the ncbi download with the -genes argument? It would be better if we could be consistent. We could also show the "ncbi download" method for human, "hgnc" method for human, "ncbi download" method for mouse, and hgnc method for mouse ... so users know all the possibilities.

Reply:
@yichaoS maybe we have decided that we are phasing out ncbi updates altogether?  So that would explain why we are hiding the ncbi human update feature from users.

Reply:
@sheridancbio Yes, we are not using NCBI updates altogether.

Reply:
But we are still using NCBI for mouse. Hopefully this does not confuse users [ncbi for mouse, hgnc for human] ... and hopefully we change over to using hgnc for mouse soon.